### PR TITLE
feat(movies): add create validator

### PR DESCRIPTION
**Pull request by soph-iest**
*What:** Adds validator with Joi to POST endpoint, and adds tests for this validation.

 **Why:** https://www.notion.so/lob/Validator-Unit-Tests-02c8f020566c4c69936cd3388f98a714

**Details:** Checked, passes all tests (that is, fails the checks as expected, and passes as expected if the cases are modified)

### CODE REVIEW

File: lib/plugins/features/movies/index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 exports.register = (server, options, next) => {
   server.route([
     {
       method: 'POST',
       path: '/movies',
       config: {
-        handler: (request, reply) => {
-          reply('hello world');
+        handler: (request, reply) =>
+          reply('hello world'),
```
Comment (sjl2) [inline lib/plugins/features/movies/index.js]: super nit: can you move this to above line?

**Comment by sjl2** [comment]
I think the subject of the commit message could continue to be movies here which simplifies the rest of the message.

```
feat(movies): add create validator
```

File: lib/plugins/features/movies/index.js (hunk 1 above)
Comment (soph-iest) [reply lib/plugins/features/movies/index.js]: I get that, makes it a little more clear. Done.

File: test/validators/movie.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  describe('title', () => {
+
+    it('is required', () => {
+      const payload = {};
+      const result = Joi.validate(payload, MovieValidator);
+
+      expect(result.error.details[0].path[0]).to.eql('title');
+
+      expect(result.error.details[0].type).to.eql('any.required');
```
Comment (kienan) [inline test/validators/movie.test.js]: super nit: You can probably group these lines together here and throughout! It's nice to have all the `expect`'s right next to each other!

Comment (soph-iest) [reply test/validators/movie.test.js]: Thanks! in the Notion page it's listed with a space between, so I'll fix that too.

Comment (kienan) [reply test/validators/movie.test.js]: Sweeet, ty! 🙌

**Pull request merged**